src/App.jsx

```diff
@@ -1,15 +1,14 @@
 import React from 'react';
 import './App.scss';
+import { Sum } from './components';
```

Comment:
To import in such a way, you should create `index.js` file and `export *` from related file